### DISCUSSION TITLE
Siri command to control fan

### DISCUSSION THREAD

**Discussion by tsfischer** [aarons22/homebridge-bond#139] [Q&A] [answered]
Just got my Bond in the mail today and it's working great with the Home app. However what I really want to do is turn our bedroom light (which is part of the fan) on and off with voice control.

Siri doesn't seem to like "Turn on bedroom fan light" (the device is called "bedroom fan". 

Thanks!

**Comment by aarons22** [comment, marked as answer]
I think Siri/HomeKit do a few things under the hood that are hard to pin down. Is the accessory in a room called "Bedroom"?

**Reply by tsfischer** [reply]
Thanks for the reply. The problem was I had the device called "Bedroom fan" in a room called "Master Bedroom". So what it was actually looking for was "Turn on the Master Bedroom Bedroom Fan Light" lol. But it turns out that *usually* you can just say "Bedroom Fan Light" after all. Siri is weird.

Incidentally since Bond doesn't really have a state for this light (not that it could since there's only one button on the original remote), I typically don't say "on" or "off", and it will just toggle it. Siri often answers back "The fan light is now on" when it actually turns it off, but that's ok by me as long as the device actually does what I want. It does annoy my wife though lol.
